software/kestra.yml

```diff
@@ -0,0 +1,10 @@
+name: Kestra
+website_url: https://kestra.io
+description: Kestra is an open-source orchestrator that simplifies workflow creation through a declarative YAML interface. It accommodates changes via UI, API, and CI/CD processes while keeping the orchestration logic in code.
```

Comment:
> orchestrator

What does it orchestrate?

Reply:
I'm sorry but despite your changes, I still don't fully understand what this software does. The landing page is extremely vague and doesn't help much. It looks geared towards deployment automation and ops automation. Is it similar to [rundeck](https://www.rundeck.com/)? Can you give an example use case? Wouldn't it be more relevant to https://landscape.cncf.io/?license=open-source or https://github.com/awesome-foss/awesome-sysadmin?

Reply:
It orchestrates data pipelines and tasks like ETL or ELT, it's a bit more like Airflow to create, schedule, and monitor workflows.

so more Airflow than Rundeck.

Reply:
Thanks, I will try to think of a clearer description - I think it should be closer to what you just wrote.

Reply:
> It orchestrates data pipelines and tasks like ETL or ELT, it's a bit more like Airflow to create, schedule, and monitor workflows.
> 
> so more Airflow than Rundeck.

`Coordinate data pipelines and tasks such as ETL and ELT. Creates, schedules and monitor workflows.`

This might work, rather than using orchestrate as its a bit ambiguous? 

Unless it doesnt coordinate either 😆.

Reply:
Event-driven, language-agnostic platform to create, schedule, and monitor workflows. in code. Coordinate data pipelines and tasks such as ETL and ELT.

is this version ok?

Reply:
> Event-driven, language-agnostic platform to create, schedule, and monitor workflows. in code. Coordinate data pipelines and tasks such as ETL and ELT.

I like this.

Reply:
Thanks guys!  it helps a lot to have your feedback on this!